src/aap_eda/wsapi/consumers.py

```diff
@@ -72,12 +72,31 @@ class Event(Enum):
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
 
 
+class DefaultConsumer(AsyncWebsocketConsumer):
+    """Default consumer for websocket connections.
+
+    This is the consumer to handle all the unexpected paths, it will close the
+    connection with an error message.
+    """
+
+    async def connect(self):
+        await self.accept()
+        await self.send('{"error": "invalid path"}')
```

Comment:
@Alex-Izquierdo Should we reveal this, it might be better to just close the connection